ITE/4/README.adoc

```diff
@@ -132,31 +133,32 @@ follows:
 }
 ```
 
-Similarly, a generic URI can be used to refer to other abstract entities, such as an
-SPDX element, and can be used to directly fetch the corresponding documents
-from a specified location. For in-toto metadata embedded within SPDX
-documents, the URI can also refer to entities elsewhere in the
-document. Resolving URIs that point to local entities elsewhere in the SPDX
-document is an implementation level detail, but it is clear that the URI MUST
-contain all the information necessary to identify the entity in that context.
-
-It is important to note that this ITE specifies what any generic URI MUST look like,
-and what rules the system MUST follow, but does not mandate a particular URI
-token that all conforming implementations MUST support. However, it is vital to create a formal process that introduces new URI
-tokens and details how their abstract contents are hashed. This ensures that
-different in-toto implementations do not diverge significantly. Creating
-an open process that involves the in-toto community can ensure consistency
-when hashing resources used by multiple stakeholders. Finally, such a process
-will also help ensure that the hashing of these entities is done in a correct
-and secure manner. Yet, detailing such a process is out of the scope of this
-ITE.
+Similarly, a generic URI can be used to refer to other abstract entities, such
+as an SPDX element, and can be used to directly fetch the corresponding
+documents from a specified location. For in-toto metadata embedded within SPDX
+documents, the URI can also refer to entities elsewhere in the document.
+Resolving URIs that point to local entities elsewhere in the SPDX document is an
+implementation level detail, but it is clear that the URI MUST contain all the
+information necessary to identify the entity in that context.
+
+It is important to note that this ITE specifies what any generic URI MUST look
+like, and what rules the system MUST follow, but does not mandate a particular
+URI token that all conforming implementations MUST support. However, it is vital
+to create a formal process that introduces new URI tokens and details how their
+abstract contents are hashed. This ensures that different in-toto
+implementations do not diverge significantly. Creating an open process that
+involves the in-toto community can ensure consistency when hashing resources
+used by multiple stakeholders. Finally, such a process will also help ensure
+that the hashing of these entities is done in a correct and secure manner. Yet,
+detailing such a process is out of the scope of this ITE.
```

Comment:
Echoing my later comment here, I don't feel that this is good enough. We should at least briefly discuss what "correct" and "secure" mean, and point to an example.

Reply:
As before, the process would establish a way to "register" a type of abstract resource. Using that process for each type of resource, we'd also establish the best practices to deal with their entities. I've mentioned it's contextual, and I've updated the security section with some general examples, where your other related comment was.